tools/ci-scripts/codegen-diff/diff_lib.py

```diff
@@ -39,18 +39,19 @@ def checkout_commit_and_generate(revision_sha, branch_name, targets=None):
 
 def generate_and_commit_generated_code(revision_sha, targets=None):
     targets = targets or [
-            target_codegen_client,
-            target_codegen_server,
-            target_aws_sdk,
-            target_codegen_server_python,
-            target_codegen_server_typescript
-        ]
+        target_codegen_client,
+        target_codegen_server,
+        target_aws_sdk,
+        target_codegen_server_python,
+        target_codegen_server_typescript
+    ]
     # Clean the build artifacts before continuing
     assemble_tasks = ' '.join([f'{t}:assemble' for t in targets])
     clean_tasks = ' '.join([f'{t}:clean' for t in targets])
     get_cmd_output("rm -rf aws/sdk/build")
     get_cmd_output(f"./gradlew --rerun-tasks {clean_tasks}")
     get_cmd_output(f"./gradlew --rerun-tasks {assemble_tasks}")
+    get_cmd_output(f"./gradlew --rerun-tasks {target_codegen_server_python}:stubs")
```

Comment:
I think we should put this into an if like `if target_codegen_server_python in targets:`

Reply:
Yes you are right.